source/api/commands/server.md

```diff
@@ -103,9 +101,9 @@ Adding delay can help simulate real world network latency. Normally stubbed resp
 cy.server({delay: 1500})
 ```
 
-***Prevent sending 404's to unmatched requests***
+***Send 404s on unmatched requests***
```

Comment:
This header logic was the wrong way round: the `force404` option doesn't _prevent_; rather, it enables the sending of a `404` on unmatched requests so I've changed the header to reflect that.

Reply:
👍